tests/foreman/ui/test_variables.py

```diff
@@ -180,336 +451,398 @@ def test_negative_update_type(self):
         2.  Enter an 'Invalid' default Value.
         3.  Submit the changes.
 
-        @assert: Variable is not updated with new type for invalid value.
-
-        @caseautomation: notautomated
+        @assert: Variable is not updated with new type and invalid default
+        value.
         """
+        name = gen_string('alpha')
+        initial_value = gen_string('alpha')
+        with Session(self.browser) as session:
+            make_smart_variable(
+                session,
+                name=name,
+                default_value=initial_value,
+                puppet_class=self.puppet['name'],
+            )
+            for data in invalid_sc_variable_data():
+                with self.subTest(data):
+                    self.smart_variable.update(
+                        name,
+                        key_type=data['sc_type'],
+                        default_value=data['value'],
+                    )
+                    self.assertIsNotNone(
+                        self.smart_variable.wait_until_element(
+                            common_locators['haserror'])
+                    )
+                    self.smart_variable.click(self.smart_variable.search(name))
+                    value = self.smart_variable.wait_until_element(
+                        locators['smart_variable.default_value']).text
+                    self.assertEqual(value, initial_value)
 
     @run_only_on('sat')
-    @stubbed()
-    @tier3
-    def test_negative_validate_default_value_required_checkbox(self):
```

Comment:
`Required` checkbox functionality is not supported by Smart Variables and actually it is not even present on UI